packages/viewer/tests/cypress/tests-e2e/footer.cy.ts

```diff
@@ -4,13 +4,63 @@ import { WEBMERCATOR } from '@swissgeo/coordinates'
 
 import { UIModes } from '@/store/modules/ui.store'
 
+interface LayerConfig {
```

Comment:
might make more sense to export the type that is defined in `packages/layers/src/api/internal.ts` (and change its name to `RawLayer` or `LayerConfigBackendResponse`, something in this vein) than to copy/paste it here